.github/workflows/scripts/unix-build.sh

```diff
@@ -7,19 +7,26 @@ set -ex
 [[ "$IN_DOCKER" == "true" ]] && cd taichi
 
 if [[ $OSTYPE == "linux-"* ]]; then
-  if [ ! -d ~/taichi-llvm-15 ]; then
-    pushd ~
-    if [ -f /etc/centos-release ] ; then
-      # FIXIME: prebuilt llvm15 on ubuntu didn't work on manylinux image of centos. Once that's fixed, remove this hack.
-      wget https://github.com/ailzhang/torchhub_example/releases/download/0.3/taichi-llvm-15-linux.zip
-    else
-      wget https://github.com/taichi-dev/taichi_assets/releases/download/llvm15/taichi-llvm-15-linux.zip
+  if ["$AMDGPU_TEST"]; then
```

Comment:
This is not right :(

I guess you wanted to test if `$AMDGPU_TEST` is set, please use

```bash
if [ ! -z $AMDGPU_TEST ]; then
    # do sth when set
else
    # do sth when not
fi
```

Note the spaces around the brackets, they are mandatory.

The condition expression means `if not (is-string-empty ($AMDGPU_TEST))`